docs/assets/templates/tools/template-minimum.dockstore.yml

```diff
@@ -0,0 +1,8 @@
+version: 1.2
+tools:
+  - subclass: CWL
+    primaryDescriptorPath: <String>
+    testParameterFiles: <String Array>
+    authors:
```

Comment:
We've gone back and forth on this and discussed barebones vs. minimum, but authors is not required, and it feels weird to have it as a minimum.